cmake/BuildLibSanitizers.cmake

```diff
@@ -23,36 +23,65 @@ macro(GEN_BUILD_TARGET name libsanitizer_path libfuzzer_path
 
   # Strip preinit object files in static libraries, otherwise a message
   # `.preinit_array section is not allowed in DSO` will prevent building DSO.
```

Comment:
Also, it is desired to add macOS runner to GHA workflow with testing. Otherwise, without testing, macOS support sooner or later will be broken.